### PR TITLE
added option for triggering haptic ticks when scrolling through numbers

### PR DESCRIPTION
**Pull request by hamaluik**
Added an option in the constructor: `haptics`, which defaults to true. When it is true, scrolling through the numbers will cause small vibration ticks to be triggered on the device.

### CODE REVIEW

File: lib/numberpicker.dart [hunk 1]
```diff
@@ -30,6 +31,7 @@ class NumberPicker extends StatelessWidget {
     this.zeroPad = false,
     this.highlightSelectedValue = true,
     this.decoration,
+    this.haptics = true,
```
Comment (MarcinusX) [inline lib/numberpicker.dart]: In order to provide backward compability, I believe it should default to false.
```suggestion
    this.haptics = false,
```

Comment (lzhuor) [reply lib/numberpicker.dart:34]: @MarcinusX This is very helpful .Could you please edit and merge it? Or I can create another PR to set all default values to false.

Comment (hamaluik) [reply lib/numberpicker.dart]: Just disabled by default. Sorry, I forgot about this PR!

**Comment by MarcinusX** [comment]
Thank you @hamaluik !

**Pull request merged**